src/components/info/StationMetadata/StationMetadata.js

```diff
@@ -3,102 +3,170 @@
-        maxWidth: 200,
-        accessor: station => station.histories[0].station_name,
+function StationMetadata({
+  stations, allNetworks, allVariables, ...restProps
+}) {
+  const columns = [
+    {
+      id: 'Network',
+      Header: 'Network',
+      minWidth: 80,
+      maxWidth: 100,
+      accessor: station => {
+        const network = stationNetwork(allNetworks, station);
+        return network ? network.name : '?';
       },
-      {
-        id: 'Location',
-        Header: 'Location',
-        minWidth: 120,
-        maxWidth: 200,
-        accessor: station => {
-          const hx = station.histories[0];
-          return (
-            <div>
-              {-hx.lon} W <br/>
-              {hx.lat} N <br/>
-              Elev. {hx.elevation} m
-            </div>
-          );
-        }
-      },
-      {
-        id: 'Record',
-        Header: 'Record',
-        minWidth: 100,
-        maxWidth: 200,
-        // accessor: station => 'record',
-        accessor: station => {
-          return (
-          <div>
-            {formatDate(station.min_obs_time)} to <br/>
-            {formatDate(station.max_obs_time)}
-          </div>
-        )}
-      },
-      {
-        minWidth: 80,
-        maxWidth: 100,
-        id: 'Obs Freq',
-        Header: 'Obs Freq',
-        accessor: station =>
-          FrequencySelector.valueToLabel(station.histories[0].freq),
-      },
-    ];
+    },
+    {
+      id: 'Native ID',
+      Header: 'Native ID',
+      minWidth: 80,
+      maxWidth: 100,
+      accessor: 'native_id'
+    },
+    {
+      id: 'Unique Names',
+      Header: 'Unique Names',
+      minWidth: 120,
+      maxWidth: 200,
+      accessor: uniqStationNames,
+      sortMethod: lexCompare,
+      Cell: row => (
+        <ul className={"compact"}>
+          {map(name => (<li>{name}</li>), row.value)}
+        </ul>
+      ),
+    },
+    {
+      id: 'Unique Locations',
+      Header: 'Unique Locations',
+      minWidth: 120,
+      maxWidth: 200,
+      sortable: false,
+      accessor: uniqStationLocations,
+      Cell: row => (
+        <ul className={"compact"}>
+          {
+              map(loc => (
+                <li>
+                  {-loc.lon} W <br/>
+                  {loc.lat} N <br/>
+                  Elev. {loc.elevation} m
+                </li>
+              ), row.value)
+          }
+        </ul>
+      ),
+    },
+    {
+      id: 'Unique Records',
+      Header: 'Unique Records',
+      minWidth: 100,
+      maxWidth: 100,
+      sortable: false,
+      accessor: uniqStationObsPeriods,
+      Cell: row => (
+        <ul className={"compact"}>
+          {
+              map(period => (
+                <li>
+                  {formatDate(period.min_obs_time)} to <br/>
+                  {formatDate(period.max_obs_time)}
+                </li>
+              ), row.value)
+          }
+        </ul>
+      ),
+    },
+    {
+      minWidth: 80,
+      maxWidth: 100,
+      id: 'Uniq Obs Freqs',
```

Comment:
What exactly does the "Uniq" in this label refer to?

Reply:
Stations can have multiple histories, hence multiple freqs; these are reduced to their unique elements (see also comment above).